Proposals/XXXX-a-beginners-guide-to-swift-contributions.md

```diff
@@ -0,0 +1,68 @@
+# A Beginner's Guide to Contributing to Swift
+
+* Proposal: [PR#22](https://github.com/SwiftCommunityPodcast/podcast/pull/22)
+
+* Authors: [Andrew Litteken](https://github.com//AndrewLitteken)
+
+* Review Manager: Unassigned
+
+* Status: Proposal Pending
+
+* Date of Recording: TBA
+
+## Introduction
+
+There is a lot of talk about contributing to Swift. But how can you actually start? This episode is to help new members of the community learn to navigate the history of the project, understand the boundaries, find a project to work on and the best way to reach out to the community for help.
+
+## Motivation
+
+The Swift Community Podcast is an experiment and is in its infancy and one of the biggest challenges is how to effectively scale the project through adequate community engagement. A codeless open source community may present challenges that are distinc from code centric ones and so Bas may have some valuable insight into this given his experience with shepherding the Swift Weekly Brief.
+
+## Proposed contribution
+
+I propose to co-produce the episode with Garric as mentioned in issue [38](https://github.com/SwiftCommunityPodcast/podcast/issues/38). See #Roles section below for open roles.
+
+## Source compatibility
+
+This episode fits with the overall theme of the podcast in the following ways:
+- highlights how to get involved in Swift devlopment
+- a new voice looking to get involved
+- helps get more people involved in the project
+- drive contributions to another community project
+
+## Alternatives considered
+
+
+## Roles
+
+- Producers: Andrew Litteken, Garric Nahapetian
+
+- Lead Host: Open (I am willing to try this, I'm still a beginner podcaster though)
+
+- Co-hosts: Suyash Srijan (By suggestion from Robert Clegg)
+
+- Editor: open
```

Comment:
If @AndrewLitteken can edit then please feel free to take on this role. I can always do it if need be though.